config/run_adios2stream.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
```

Comment:
Is this file going in or is it just here for testing?

Reply:
@jakubcerveny you're correct, in principle this is mostly for testing and coverage information. About keeping adios2 in several examples, I was mostly using them for testing coverage within ParaView/VTK. I'll accommodate to the preference of the maintainers.